kolibri/tasks/api.py

```diff
@@ -1,25 +1,53 @@
 import logging as logger
 
+from django.apps.registry import AppRegistryNotReady
+try:
+    from django.apps import apps
+    apps.check_apps_ready()
+except AppRegistryNotReady:
+    import django
+    django.setup()
+
 import requests
+import platform
 from django.core.management import call_command
 from django.http import Http404
 from django.utils.translation import ugettext as _
-from django_q.models import OrmQ, Task
-from django_q.tasks import async
+from django_q.humanhash import uuid
 from kolibri.content.models import ChannelMetadataCache
 from kolibri.content.utils.channels import get_mounted_drives_with_channel_info
 from kolibri.content.utils.paths import get_content_database_file_url
 from rest_framework import serializers, viewsets
 from rest_framework.decorators import list_route
 from rest_framework.response import Response
+from django_q.tasks import async
+from django_q.models import Task, OrmQ
+
+from multiprocessing import Process
 
 logging = logger.getLogger(__name__)
 
+def windows_handle_async_call(target_func, *args, **kwargs):
```

Comment:
Can you add docstrings on these two functions, explaining why we had to do it this way for Windows?